domain/src/TaxYear/Services/TaxYearGenerator.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Domain\TaxYear\Services;
+
+final class TaxYearGenerator
+{
+    public static function fromYear(int $year): string
+    {
+        return sprintf('%s-%s', $year, $year + 1);
+    }
+}
```

Comment:
ℹ️ Moved this to a service as it's used if different places now, but then again only the `TaxYear` aggregate root should have to deal with this. Will refactor eventually